cmd/rook/ceph/operator.go

```diff
@@ -57,6 +57,7 @@ func init() {
 	operatorCmd.Flags().StringVar(&csi.CSIParam.ProvisionerImage, "csi-provisioner-image", csi.DefaultProvisionerImage, "csi provisioner image")
 	operatorCmd.Flags().StringVar(&csi.CSIParam.AttacherImage, "csi-attacher-image", csi.DefaultAttacherImage, "csi attacher image")
 	operatorCmd.Flags().StringVar(&csi.CSIParam.SnapshotterImage, "csi-snapshotter-image", csi.DefaultSnapshotterImage, "csi snapshotter image")
+	operatorCmd.Flags().BoolVar(&csi.CSIParam.AllowUnsupported, "csi-allow-unsupported-image", false, "csi allow unsupported image versions")
```

Comment:
Note: @travisn this can affect downstream builds I think as the image tag is different compare to upstream

Reply:
@Madhu-1 Are you saying the downstream CSI images would have a downstream version instead of also including the upstream version? I was assuming we would follow ceph's pattern, which includes the upstream version.

Reply:
yes currently it's having a different version

Reply:
lets not added its argument, we can make use of ENV like https://github.com/rook/rook/blob/master/pkg/operator/ceph/csi/spec.go#L214

Reply:
Why would we not also allow this to be an arg?

Reply:
if I recollect things correctly while reviewing some PR @leseb told we should stop adding more arguments to operator main(init) function.

if am wrong we can add as an argument here

Reply:
@leseb What do you prefer?

Reply:
It wasn't me :)